src/bp-core/classes/class-bp-post-notification.php

```diff
@@ -0,0 +1,211 @@
+<?php
+/**
+ * BuddyBoss Post Notification Class.
+ *
+ * @package BuddyBoss\Core
+ *
+ * @since BuddyBoss [BBVERSION]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Set up the BP_Post_Notification class.
+ *
+ * @since BuddyBoss [BBVERSION]
+ */
+class BP_Post_Notification extends BP_Core_Notification_Abstract {
```

Comment:
please rename the class name to `BB_Post_Notification` and also the file name s`class-bb-post-notification.php`